certs/control-api/ca.crt

```diff
@@ -0,0 +1,74 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I would not block this PR on this request, because it might complicate things unnecessarily. 

That said, as a team we need to find a way avoid checking in certificates. If we need certs, we should generate them at runtime for test scenarios, or provide instructions for users to generate them at "build time" into a gitignore'd directory.

Terraform has a module for doing this https://registry.terraform.io/providers/hashicorp/tls/latest/docs/resources/self_signed_cert

I would open a follow up issue so we can fix this up later.

Reply:
@sofiashalotenko have we ever attempted to generate certs with TF? I'm just looking for any reason it _wouldn't_ work, in principle.

Reply:
issue filed https://github.com/greymatter-io/terraform-greymatter-ecs/issues/23